src/basic.h

```diff
@@ -29,7 +29,7 @@ namespace SymEngine {
 
 class Visitor;
 class Symbol;
-
+//class Polynomial;
```

Comment:
@sushant-hiray It is not required. It seems to be left-over from when I was debugging.